src/utils/helpers/structure/generateReviewSectionActions.ts

```diff
@@ -131,10 +131,13 @@ const generateReviewSectionActions: GenerateSectionActions = ({
 
     const totalNewReviewable = section?.consolidationProgress?.totalNewReviewable
 
+    const reviewStatus = thisReview?.current.reviewStatus
+
     const isReviewable =
-      (totalReviewable || 0) > 0 &&
       assignmentStatus === ReviewAssignmentStatus.Assigned &&
-      assignedSections.includes(section.details.code)
+      assignedSections.includes(section.details.code) &&
+      // Only check totalReviewable if review already exists
+      (reviewStatus ? (totalReviewable || 0) > 0 : true)
```

Comment:
We might get away with removing the `(totalReviewable || 0) > 0` check completely, but I kept it in just in case -- but only if the review exists